.travis.yml

```diff
@@ -35,6 +50,18 @@ matrix:
       env: TOXENV=py35-df20-django20-drf37
     - python: 3.5
       env: TOXENV=py35-df20-django20-drf38
+    - python: 3.5
+      env: TOXENV=py35-df20-django20-drf39
+    - python: 3.5
+      env: TOXENV=py35-df20-django20-drfmaster
+    - python: 3.5
+      env: TOXENV=py35-df20-django21-drf37
```

Comment:
Django 2.1 is only supported with DRF 3.9 so we should remove this.

Reply:
You don't think it's a good idea to test master and get an early heads-up (e.g. on dot-releases of 3.9)?